www/README.md

```diff
@@ -0,0 +1,35 @@
+# harp.gl website
+
+Source for [harp.gl](https://heremaps.github.io/harp.gl) "landing" page
+
+
+## Local testing/development
+
+* `yarn start --open` - starts `webpack-dev-server` for this project
+
+* `yarn build` -> build into `dist/` ->
+
+## Production architecture
+
+Main part of "app" is hosted on [Heremaps org Github Pages](https://heremaps.github.io/harp.gl).
+
+Examples and docs are hosted on S3, behind Cloudfront.
+
+(Looks like) Cloudfront is the frontend for `harp.gl`, works as proxy for
+1) `https://www.harp.gl/docs/*` - rev-proxied to S3
+2) rest - rev-prexied to `https://heremaps.github.io/harp.gl`
+
+## Production Deployment
+
+(all paths relative to `harp.gl` project root)
+
+1) `harp.gl` site is static and lives in `docs/index/html`. It uses `harp.gl` bundle and three.js
```

Comment:
That's (hopefully)  not true anymore (I mean using the harp.gl bundle from unpkg.com)

Reply:
Yep, ... it's just doc in progress - i was documenting how it worked. This patch fixes it, i will remove outdated WIP docs.